compiler/daml-lf-tools/src/DA/Daml/LF/TypeChecker/Serializability.hs

```diff
@@ -53,6 +53,10 @@ serializabilityConditionsType world0 _version mbModNameTpls vars = go
       TList typ -> go typ
       TOptional typ -> go typ
       TMap typ -> go typ
+      TNumeric n -> go n
+        -- TODO (#2289): decide if n should be forced to be a static nat
+        -- literal within LF's bounds for the numeric type (0 <= n <= 38).
```

Comment:
In the first delivered version, we will require the `Numeric` to be applied to a static `nat`. 
This simplified a lot of work on the runtime side. 
For instance, dealing with type variables in java/scala codegen is not trivial at all.  
We may relax this restriction in the future if required.

Reply:
This seems like a good idea, and lets us check the bound here explicitly.

Reply:
I agree. Let's keep it tight at the beginning and loosen it when there's demand.

Reply:
Ok! I added a check to make this fixed number between 0 and 38 for now.